modules/simple_forms_api/spec/models/vha_10_7959f_2_spec.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+folder_path = 'modules/simple_forms_api/spec/fixtures/test_file/'
+file_name = 'test_file.pdf'
+file_path = File.join(folder_path, file_name)
+
+RSpec.describe SimpleFormsApi::VHA107959f2 do
+  let(:data) do
+    {
+      'veteran' => {
+        'full_name' => { 'first' => 'John', 'middle' => 'P', 'last' => 'Doe' },
+        'va_claim_number' => '123456789',
+        'mailing_address' => { 'postal_code' => '12345' }
+      },
+      'form_number' => '10-7959F-2',
+      'veteran_supporting_documents' => [
+        { 'confirmation_code' => 'abc123' },
+        { 'confirmation_code' => 'def456' }
+      ]
+    }
+  end
+  let(:vha107959f2) { described_class.new(data) }
+
+  describe '#metadata' do
+    it 'returns metadata for the form' do
+      metadata = vha107959f2.metadata
+
+      expect(metadata).to include(
+        'veteranFirstName' => 'John',
+        'veteranMiddleName' => 'P',
+        'veteranLastName' => 'Doe',
+        'fileNumber' => '123456789',
+        'zipCode' => '12345',
+        'source' => 'VA Platform Digital Forms',
+        'docType' => '10-7959F-2',
+        'businessLine' => 'CMP'
+      )
+    end
+  end
+
+  describe '#handle_attachments' do
```

Comment:
This references `CombinePDF` but I don't see that being used. If it is being used then please ignore this comment.

Reply:
@stevenjcumming I removed this test and updated how it will work in the new module I created as this code will eventually be obsolete after ivc_champva/ is used